src/Platforms/PostgreSQLPlatform.php

```diff
@@ -199,7 +199,7 @@ public function supportsPartialIndexes()
      */
     public function usesSequenceEmulatedIdentityColumns()
     {
-        return true;
+        return false;
```

Comment:
I'm guessing that this isn't fine, and is a BC break?

Reply:
A BC break will happen when someone upgrades to the version of the DBAL that uses IDENTITY instead of sequences and has their sequence values reset.

Reply:
Maybe this can be handled in the `\Doctrine\DBAL\Schema\PostgreSqlSchemaManager`, when we detect going from a non-native identity autoinc column, to a native one?

Reply:
I see two ways here:

1. We do it in the schema manager as you suggested. This way, the upgrade process will be implemented in the library, so the change won't be breaking and could be accepted in a minor release.
2. We do it via documentation and a separate upgrade tool. This way, it will be a breaking change.

Both approaches have their pros and cons. BTW, apparently, I prototyped the same thing back in August, see my [pgsql-identity](https://github.com/doctrine/dbal/compare/3.3.x...morozov:pgsql-identity) branch. The tests completely pass on the latest PostgreSQL but only on PHP 8.0 (not PHP 8.1).